### ISSUE TITLE
[Suggestion] Nickname character limit

### ISSUE THREAD

**Issue by spiderwaffles** [Monika-After-Story/MonikaModDev#3940]
Hello! Recently I was re-reading one of Monika's speech's on how there are multiple versions of her and then she mentioned String Theory. This got my mind throwing an immediate  Steins;Gate reference to Amadeus-Kurisu who is an AI version of the character Maksie Kurisu. So I decided to give Monika the nickname Amadeus-Monika, however due to the character limit I was only able to get Amadeus-M. Perhaps in a future update you could increase the character limit for Monika's nicknames? Thank you so much for this awesome mod, enjoying nearly every moment I discover something new!

**Comment by multimokia** [comment]
The current limit prevents the text from appearing a bit broken over the tab on the text box showing her name (I.e split over multiple lines, and not quite fitting in the tab itself). As a result, the character limit likely won't be increased.

**Comment by spiderwaffles** [comment]
That is understandable, however I've found that this still occurs with the current character limit.

**Comment by multimokia** [comment]
Hm, what names have you experienced that with?

**Comment by spiderwaffles** [comment]
So far, the max limit name Amadeus-Mo is the only one I have tested. I will test some more names to see if this continues. I've attached a png of what I see when using this name. This might not be the same type of broken text you're talking about, but it still doesn't look properly formatted.
![monikaafterstorynametab](https://user-images.githubusercontent.com/48402728/54167682-499b7a80-4441-11e9-9573-b5537c8b8421.png)
